client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/BackfillaParameters.kt

```diff
@@ -0,0 +1,105 @@
+package app.cash.backfila.client.misk.internal
+
+import app.cash.backfila.client.misk.Backfill
+import app.cash.backfila.client.misk.BackfillConfig
+import app.cash.backfila.protos.clientservice.GetNextBatchRangeRequest
+import app.cash.backfila.protos.clientservice.PrepareBackfillRequest
+import app.cash.backfila.protos.clientservice.RunBatchRequest
+import app.cash.backfila.protos.service.Parameter
+import com.google.inject.TypeLiteral
+import com.squareup.moshi.Types
+import java.lang.reflect.ParameterizedType
+import kotlin.reflect.KClass
+import kotlin.reflect.KParameter
+import kotlin.reflect.full.memberProperties
+import kotlin.reflect.full.primaryConstructor
+import kotlin.reflect.jvm.jvmErasure
+import misk.exceptions.BadRequestException
+import okio.ByteString
+import okio.ByteString.Companion.encodeUtf8
+
+inline fun <reified T : Any> getBackfilaParametersForBackfill(backfillClass: KClass<Backfill<*, *, T>>): List<Parameter> {
```

Comment:
make everything in this file internal

Reply:
although you could move this to `client` and reuse it for the non-misk client?

Reply:
I made everything internal. Maybe we can convert this over? Problem for another day.